osoby2.csv

```diff
@@ -0,0 +1,57 @@
+;Priezvisko;Meno;Email;Stredn� �kola;Stredn� �kola (adresa);Bydlisko - ulica;PS�;Bydlisko - obec
```

Comment:
Is encoding wrong on our side, or is it GitHub?

Reply:
not sure